templates/ntp.conf.erb

```diff
@@ -34,8 +36,14 @@ interface listen <%= interface %>
 broadcastclient
 <% end -%>
 
+# Set up servers for ntpd with next options:
+# server - IP address or DNS name of upstream NTP server
+# iburst - allow send sync packages faster if upstream unavailable
+# prefer - select preferrable server
+# minpoll - set minimal update frequency
+# maxpoll - set maximal update frequency
 <% [@servers].flatten.each do |server| -%>
-server <%= server %><% if @iburst_enable == true -%> iburst<% end %><% if @preferred_servers.include?(server) -%> prefer<% end %>
+server <%= server %><% if @iburst_enable == true -%> iburst<% end %><% if @preferred_servers.include?(server) -%> prefer<% end %><% if @minpoll.to_i != 6 -%> minpoll <%= @minpoll %><% end %><% if @maxpoll.to_i != 10 -%> maxpoll <%= @maxpoll %><% end %>
```

Comment:
If you want these options to only be set if not the default value, I'd rather see them default to undef rather than having the defaults hardcoded in the template.